signingscript/test/helpers/task_generator.py

```diff
@@ -0,0 +1,34 @@
+import os
+import json
+
+
+class TaskGenerator(object):
+    def __init__(self, urls=None):
+        self.urls = urls if urls is not None else [
+            'https://queue.taskcluster.net/v1/task/ZB9X1cvCRo-hTps7BDk8pw/artifacts/\
+public%2Fbuild%2Ffirefox-52.0a1.en-US.win64.installer.exe'
+        ]
+
+    def generate_file(self, work_dir):
+        task_file = os.path.join(work_dir, 'task.json')
+        with open(task_file, 'w') as f:
+            json.dump(self.generate_json(), f)
+        return task_file
+
+    def generate(self):
```

Comment:
All this function needs 'self' for is to provide self.urls, which could be a function kwarg, e.g.

`def generate_task(urls=('https://queue.taskcluster.net/v1/task/ZB9X1cvCRo-hTps7BDk8pw/artifacts/\
+public%2Fbuild%2Ffirefox-52.0a1.en-US.win64.installer.exe', )):`